help/develop/getting-started.md

```diff
@@ -3,208 +3,215 @@
 
 # Getting Started with Building on Datagrok
 
-Welcome to [Datagrok](https://www.datagrok.ai), the next generation data platform!
```

Comment:
@AndSviat why do we prefer to remove the hyperlink?

Reply:
The hyperlinks are moved from the text to the bottom of the document. Markdown knows how to map the links. I move all the links to the bottom of Markdown docs for readability. It's very difficult to read the text when there are so many links, and you have to jump constantly between words, skipping links. 

Also, when you have two or three links that are the same, it's easier to add the link once at the bottom of the doc, like this: 
`[Datagrok]: https://www.datagrok.ai.`

Then, throughout the document, we can add just the `[Datagrok]` part many times, but the link is the same.